plugins/cypher-language/src/main/java/org/polypheny/db/cypher/CypherProcessor.java

```diff
@@ -152,7 +154,79 @@ public AlgDataType getParameterRowType( Node left ) {
 
     @Override
     public List<String> splitStatements( String statements ) {
-        return Arrays.stream( statements.split( ";" ) ).filter( q -> !q.trim().isEmpty() ).toList();
```

Comment:
Actually `CypherParser.jj` supports multiple statements, (`CypherParser::parseStmts` even returns a list of `CypherStatement`s), so this should somehow be used directly if possible.